### PR TITLE
Suppress repetition in getInstructorFeedbackSessionPublishAndUnpublishAction in PageData and InstructorFeedbackResultsPageData #3796

### LINKED ISSUE #3796: Suppress repetition in getInstructorFeedbackSessionPublishAndUnpublishAction in PageData and InstructorFeedbackResultsPageData
Found this while doing #3787. Right now both are almost identical. The differences are:
1. The button class (`btn-default btn-xs` vs `btn-primary btn-block`), which can be abstracted out.
2. The togglePublishEvaluation parameters (a js function), I believe the one in FeedbackResults is lacking the second parameter.
3. The one in PageData is wrapped in div wrapper, which is unnecessary.
4. (Un)Publish Results vs (Un)Publish results, which is easy to sort out.

This will be done on master and needs to be merged to JSTL-Dev first before I can continue with #3787.

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #3796 
1. The change to the functional code amounts to only 23 additions and 50 deletions so reviewing this should not be hard.
2. There is a hell lot of deletion of (erroneously-)generated comment block in Results HTML. How they are generated in the first place, I don't know.
3. The Download Results modification is not actually part of the issue, but I did it anyway because in this page (the two buttons on the extreme right):
sentencevstitle
it feels really weird if one is in Title Case and the other is in Sentence case.
Hoping to get this merged fast so that I can work on #3787

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@bamboo3250 do you have time to review this now? :smiley:

**Comment by bamboo3250** [comment]
@wkurniawan07 Okay, I'm reviewing it now.

**Comment by bamboo3250** [comment]
Sorry to make you wait. The change seems fine :+1:

**Comment by wkurniawan07** [comment]
It's ok, thanks for reviewing :+1:

File: src/test/resources/pages/instructorFeedbackResultsSortQuestion.html [hunk 1]
```diff
@@ -2710,114 +2710,6 @@
...
-                        <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
```
Comment (damithc) [inline src/test/resources/pages/instructorFeedbackResultsSortQuestion.html]: These line are caused by a bug? How did it ever get merged?

Comment (wkurniawan07) [inline src/test/resources/pages/instructorFeedbackResultsSortQuestion.html]: I did an investigation and managed to find out why. This is not a bug, but rather an unfortunate series of event. To explain, let's get back to about three weeks ago.
I made this commit: ac68b6c, which basically is running God mode through one UI test and getting one whole bunch of test files updated with that comment line. Up to here it still looks fine.
Then this commit is made: ba08c70. Looking at the issue being addressed, I understand that a large amount of line is removed.
The sum of addition of the comment lines and removal of those existing lines is what we see up there.

Comment (damithc) [inline src/test/resources/pages/instructorFeedbackResultsSortQuestion.html]: I see. Anyway, all is fine now right?

Comment (wkurniawan07) [inline src/test/resources/pages/instructorFeedbackResultsSortQuestion.html]: Yeah. I have run God mode once again in this PR, which removed all of them.

**Pull request merged**